selfdrive/ui/paint.cc

```diff
@@ -14,30 +14,26 @@
 #include "paint.hpp"
 #include "sidebar.hpp"
 
-
 // TODO: this is also hardcoded in common/transformations/camera.py
 // TODO: choose based on frame input size
 #ifdef QCOM2
 const float y_offset = 150.0;
 const float zoom = 1.1;
-const mat3 intrinsic_matrix = (mat3){{
-  2648.0, 0.0, 1928.0/2,
-  0.0, 2648.0, 1208.0/2,
-  0.0,   0.0,   1.0
-}};
+const mat3 intrinsic_matrix = (mat3){{2648.0, 0.0, 1928.0 / 2,
```

Comment:
openpilot does not include a formatter in this pre-commit. shall we just use clang-format?